src/main/fc/rc_modes.c

```diff
@@ -154,7 +154,7 @@ void updateActivatedModes(void)
         updateMasksForMac(mac, &andMask, &newMask, bActive);
     }
 
-    bitArrayXor(&newMask, sizeof(&newMask), &newMask, &andMask);
```

Comment:
The only problem is here - `sizeof(&newMask)` is size of pointer to newmask - 4 bytes.
`sizeof(newMask)` will fix the problem